[2025-09-02T02:32:10.119+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-02T02:32:10.227+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ecommerce_pipeline.sales_per_country scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-09-02T02:32:10.254+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ecommerce_pipeline.sales_per_country scheduled__2025-09-01T00:00:00+00:00 [queued]>
[2025-09-02T02:32:10.267+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-09-02T02:32:10.332+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): sales_per_country> on 2025-09-01 00:00:00+00:00
[2025-09-02T02:32:10.346+0000] {standard_task_runner.py:63} INFO - Started process 609 to run task
[2025-09-02T02:32:10.357+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'ecommerce_pipeline', 'sales_per_country', 'scheduled__2025-09-01T00:00:00+00:00', '--job-id', '161', '--raw', '--subdir', 'DAGS_FOLDER/ecommerce_pipeline_dag.py', '--cfg-path', '/tmp/tmpvxh26qxv']
[2025-09-02T02:32:10.362+0000] {standard_task_runner.py:91} INFO - Job 161: Subtask sales_per_country
[2025-09-02T02:32:10.422+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.10/site-packages/***/settings.py:195 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-02T02:32:10.601+0000] {task_command.py:426} INFO - Running <TaskInstance: ecommerce_pipeline.sales_per_country scheduled__2025-09-01T00:00:00+00:00 [running]> on host 6aba05baf296
[2025-09-02T02:32:10.919+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='arel' AIRFLOW_CTX_DAG_ID='ecommerce_pipeline' AIRFLOW_CTX_TASK_ID='sales_per_country' AIRFLOW_CTX_EXECUTION_DATE='2025-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-01T00:00:00+00:00'
[2025-09-02T02:32:10.921+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-02T02:32:10.967+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-02T02:32:10.971+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'PGPASSWORD=*** psql -h postgres -U *** -d *** -f /opt/***/sql/sales_per_country.sql']
[2025-09-02T02:32:11.001+0000] {subprocess.py:86} INFO - Output:
[2025-09-02T02:32:11.502+0000] {subprocess.py:93} INFO - psql:/opt/***/sql/sales_per_country.sql:1: NOTICE:  table "sales_per_country" does not exist, skipping
[2025-09-02T02:32:11.503+0000] {subprocess.py:93} INFO - DROP TABLE
[2025-09-02T02:32:11.505+0000] {subprocess.py:93} INFO - psql:/opt/***/sql/sales_per_country.sql:10: ERROR:  relation "transactions" does not exist
[2025-09-02T02:32:11.509+0000] {subprocess.py:93} INFO - LINE 6: FROM transactions
[2025-09-02T02:32:11.511+0000] {subprocess.py:93} INFO -              ^
[2025-09-02T02:32:11.535+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-02T02:32:11.538+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-02T02:32:11.684+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=ecommerce_pipeline, task_id=sales_per_country, run_id=scheduled__2025-09-01T00:00:00+00:00, execution_date=20250901T000000, start_date=20250902T023210, end_date=20250902T023211
[2025-09-02T02:32:11.777+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-02T02:32:11.852+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-09-02T02:32:11.863+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
